src/test/enginesynctest.cpp

```diff
@@ -1429,3 +1429,52 @@ TEST_F(EngineSyncTest, ZeroBpmNaturalRate) {
     EXPECT_EQ(0.0,
               ControlObject::getControl(ConfigKey(m_sGroup1, "local_bpm"))->get());
 }
+
+TEST_F(EngineSyncTest, SyncPhaseToIfQuantize) {
```

Comment:
how about "QuantizeImpliesSyncPhase"  (in your wording, it should be "too")

Reply:
ping